tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -6094,7 +6099,7 @@ def get_matmul(arr):
         from numpy import matmul, shape
         a = matmul(arr, arr)
         s = shape(a)
-        return len(s), s[0], s[1], a[0,1], a[1,0]
+        return len(s) + s[0] + s[1] + a[0,1] + a[1,0]
```

Comment:
Can you explain this change please

Reply:
ah i missed this, it was for testing with np.isclose without .any() to .all().